### PR TITLE
Async resource loading

### PR DESCRIPTION
**Pull request by agitator**

### CODE REVIEW

**Comment by agitator** [comment]
@jenkins-plone-org please run jobs

**Comment by agitator** [comment]
@jenkins-plone-org please run jobs

**Comment by agitator** [comment]
@jenkins-plone-org please run jobs

File: src/plone/staticresources/static/plone-base.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+// You should have received a copy of the GNU General Public License along with
+// this program; if not, write to the Free Software Foundation, Inc., 51
+// Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+//
+
+if (window.jQuery) {
+  define('jquery', [], function () {
+    'use strict';
+    return window.jQuery;
+  });
```
Comment (davilima6) [inline src/plone/staticresources/static/plone-base.js]: I'm not sure what the block above does (a comment would be nice, is it turning global jquery into a requirejs module?).

Do we need it both here and in plone.js?

File: src/plone/staticresources/static/plone-base.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    (window.frameElement.nodeName === 'IFRAME')) {
+    $(document).ready(function () {
+      $('body').addClass('pat-plone');
+      if (!registry.initialized) {
+        registry.init();
+      }
+    });
+  }
+
+  // TODO: Needs to be moved to controlpanel js
```
Comment (davilima6) [inline src/plone/staticresources/static/plone-base.js]: do you know if we have an issue for this TODO already?

**Comment by davilima6** [comment]
I'm not sure the Jenkins failures is related. I'd run it again.

File: src/plone/staticresources/static/plone-base.js (hunk 2 above)
Comment (agitator) [reply src/plone/staticresources/static/plone-base.js]: no idea

File: src/plone/staticresources/static/plone-base.js (hunk 1 above)
Comment (agitator) [reply src/plone/staticresources/static/plone-base.js]: The block does exactly that.
Removed the doubled definition from plone.js locally and still seems to work

**Comment by agitator** [comment]
@jenkins-plone-org please run jobs

**Comment by agitator** [comment]
@jenkins-plone-org please run jobs

**Comment by thet** [comment]
@davilima6 the aggregate bundles are in no-async, no-defer mode by intention. therefore we unset the ``merge_with`` attribute for the plone and plone-logged-in bundles. With http2 delivering more resources shouldn't be that much of a problem....

**Pull request merged**